### PR TITLE
🌐 Add Portuguese translation for `docs/pt/docs/advanced/websockets.md`

### PR DESCRIPTION
**Pull request by devfernandoa**

### CODE REVIEW

**Review by nazarepiedady — APPROVED** [review]
You did great work. Thanks for your collaboration.

**Comment by nazarepiedady** [comment]
@ceb10n and @cassiobotaro, which one of you have time to review this pull request?

File: docs/pt/docs/advanced/websockets.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+# WebSockets
+
+Você pode usar  <a href="https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API" class="external-link" target="_blank">WebSockets</a> com **FastAPI**.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:3]: There are two spaces here:
Você pode usar  <a href...

Just remove one so it doesn't look strange :D

File: docs/pt/docs/advanced/websockets.md [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+
+## Criando um `websocket`
+
+Em sua aplicação **FastAPI**, crie um `websocket`:
+
+```Python hl_lines="1  46-47"
+{!../../docs_src/websockets/tutorial001.py!}
+```
+
+/// note | "Detalhes Técnicos"
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:53]: You can remove the double quotes here:

```
/// note | Detalhes Técnicos
```

File: docs/pt/docs/advanced/websockets.md [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+
+Você também poderia usar `from starlette.websockets import WebSocket`.
+
+A **FastAPI** fornece o mesmo `WebSocket` diretamente apenas como uma conveniência para você, o desenvolvedor. Mas ele vem diretamente do Starlette.
+
+///
+
+## Aguardar por mensagens e enviar mensagens
+
+Em seu rota WebSocket você pode `await` por mensagens e enviar mensagens.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:63]: change to:

Em **sua** rota...

And I think it's a little strange: você pode `await` por mensagens

In english it makes sense, but when translating, it loses the sense...

I think you can translate using something like:

Em sua rota WebSocket você pode esperar (`await`) por mensagens e enviar mensagens.

File: docs/pt/docs/advanced/websockets.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+
+```console
+$ fastapi dev main.py
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+Abra seu broser em: <a href="http://127.0.0.1:8000" class="external-link" target="_blank">http://127.0.0.1:8000</a>.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:85]: missed the **w** in browser: Abra seu bro**w**ser em...

But I think you can use:

Abra o seu navegador em...

File: docs/pt/docs/advanced/websockets.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+Nos endpoints WebSocket você pode importar do `fastapi` e usar:
+
+* `Depends`
+* `Security`
+* `Cookie`
+* `Header`
+* `Path`
+* `Query`
+
+They work the same way as for other FastAPI endpoints/*path operations*:
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:116]: This line needs to be removed :D

File: docs/pt/docs/advanced/websockets.md [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+Eles funcionam da mesma forma que para outros endpoints FastAPI/*operações de rota*:
+
+//// tab | Python 3.10+
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:119]: You can replace all the ```Python... with:

{* ../../docs_src/websockets/tutorial002_an_py310.py  hl[68:69,82] *}

See this issue for reference: Update docs include syntax for source examples https://github.com/fastapi/fastapi/issues/12554

File: docs/pt/docs/advanced/websockets.md [hunk 7]
```diff
@@ -0,0 +1,257 @@
...
+
+A **FastAPI** fornece o mesmo `WebSocket` diretamente apenas como uma conveniência para você, o desenvolvedor. Mas ele vem diretamente do Starlette.
+
+///
+
+## Aguardar por mensagens e enviar mensagens
+
+Em seu rota WebSocket você pode `await` por mensagens e enviar mensagens.
+
+```Python hl_lines="48-52"
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:65]: Here you can use the new style for includes:

{* ../../docs_src/websockets/tutorial001.py hl[48:52] *}

See this issue for reference: Update docs include syntax for source examples https://github.com/fastapi/fastapi/issues/12554

File: docs/pt/docs/advanced/websockets.md [hunk 8]
```diff
@@ -0,0 +1,257 @@
...
+
+///
+
+```Python hl_lines="68-69  81"
+{!> ../../docs_src/websockets/tutorial002.py!}
+```
+
+////
+
+/// info
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:171]: You can add translation for the admonition here:

```
/// info | Informação
```

File: docs/pt/docs/advanced/websockets.md [hunk 9]
```diff
@@ -0,0 +1,257 @@
...
+
+```console
+$ fastapi dev main.py
+
+<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+```
+
+</div>
+
+Abrar seu browser em: <a href="http://127.0.0.1:8000" class="external-link" target="_blank">http://127.0.0.1:8000</a>.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:193]: You can translate browser here (and remove the r in Abra):

Abra o seu navegador em: ...

File: docs/pt/docs/advanced/websockets.md [hunk 10]
```diff
@@ -0,0 +1,257 @@
...
+
+Com isso você pode conectar o WebSocket e então enviar e receber mensagens:
+
+<img src="/img/tutorial/websockets/image05.png">
+
+## Lidando com desconexões e múltiplos clientes
+
+Quando uma conexão WebSocket é fechada, o `await websocket.receive_text()` levantará uma exceção `WebSocketDisconnect`, que você pode então capturar e lidar como neste exemplo.
+
+//// tab | Python 3.9+
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:214]: Here you can replace all the code with:

{* ../../docs_src/websockets/tutorial003_py39.py hl[79:81] *}

See this issue for reference: Update docs include syntax for source examples https://github.com/fastapi/fastapi/issues/12554

File: docs/pt/docs/advanced/websockets.md [hunk 11]
```diff
@@ -0,0 +1,257 @@
...
+
+```Python hl_lines="81-83"
+{!> ../../docs_src/websockets/tutorial003.py!}
+```
+
+////
+
+Para testar:
+
+* Abrar o aplicativo com várias abas do navegador.
```
Comment (ceb10n) [inline docs/pt/docs/advanced/websockets.md:232]: remove the **r**:

Abra o aplicativo...

**Review by ceb10n — CHANGES_REQUESTED** [review]
Thank you @devfernandoa 

Just some typos and suggestions

And it would be nice to use the new style for includes described in the issue: Update docs include syntax for source examples https://github.com/fastapi/fastapi/issues/12554

**Review by ceb10n — APPROVED** [review]
That was fast! ⚡

**Comment by alejsdev** [comment]
Thanks for your contribution! @devfernandoa :tada: 
And thanks for the reviews and suggestions @ceb10n :mag:

**Pull request merged**